client/src/App.js

```diff
@@ -1,16 +1,26 @@
 import React from "react";
 import Shorties from "./pages/Shorties";
 import styled from "styled-components/macro";
+import logoSrc from "./assets/logo.svg";
```

Comment:
How about this? Then you can style the svg directly and don't need an img.
```suggestion
import { ReactComponent as LogoSrc } from "./assets/logo.svg";
```

Reply:
Nice one! 
This depends on the use case. If I load it as `img`, the browser can cache it. The image will be loaded asynchronously. 

If I convert it to a React component, it will be added to the JavaScript bundle. In this case, it will be displayed as soon as it renders.

Reply:
Understood :) Thank you for explaining!